package.json

```diff
@@ -56,6 +56,7 @@
     "html5-qrcode": "^2.3.8",
     "i18next": "^22.4.15",
     "i18next-browser-languagedetector": "^7.0.1",
+    "liquidjs-lib": "^6.0.2-liquid.28",
```

Comment:
To review: dependencies in this package

I think we do not want to use `tiny-secp256k1` if possible, not `ecpair` either

Reply:
@rolznz, pushed an adapter module for @noble/secp256k1 [here - bfea5e4](https://github.com/getAlby/lightning-browser-extension/pull/2516/commits/bfea5e44e4867fb62508fdb434cd2f402189ecdc). Liquid code does not depend anymore on `tiny-secp256k1` neither `ecpair` but I have to wait for merge to totally remove them from deps

Reply:
@louisinger awesome!